compass/parallel.py

```diff
@@ -37,65 +38,42 @@ def get_available_cores_and_nodes(config):
         else:
             args = ['sinfo', '--noheader', '--node', node, '-o', '%Z']
             threads_per_core = _get_subprocess_int(args)
-        cores_per_node = sockets_per_node*cores_per_socket*threads_per_core
+        cores_per_node = sockets_per_node * cores_per_socket * threads_per_core
         args = ['squeue', '--noheader', '-j', job_id, '-o', '%D']
         nodes = _get_subprocess_int(args)
         cores = cores_per_node * nodes
+        mpi_allowed = True
+    elif parallel_system == 'login':
```

Comment:
I like this solution - defining a different system for a login node seems like it would provide the flexibility needed to handle the situation properly.